dropshot_endpoint/Cargo.toml

```diff
@@ -1,8 +1,11 @@
 [package]
 name = "dropshot_endpoint"
+description = "macro used by dropshot consumers for registering handlers"
 version = "0.1.0"
 authors = ["Adam H. Leventhal <ahl@oxide.computer>"]
 edition = "2018"
+license = "Apache-2.0"
```

Comment:
I mentioned this on chat, but this means this package is GPL incompatible. Standard Rust licensing is `Apache-2.0/MIT` for this reason. I don't know how folks like @bcantrill feel about that.

Reply:
I'd suggest we decouple licensing from this PR.

It's currently documented here https://github.com/oxidecomputer/meta/blob/51da81dafc71623412f526ac1a7778e712ef044e/engineering/code.md#open-source-licensing. @steveklabnik if you think we should reconsider, I'd suggest we revisit the topic in chat. It's unlikely that we're going to get a barrage of contributors from whom we'd need to extract permission to relicense dropshot if we change our position after that discussion.

Reply:
great!